lib/data/entity.js

```diff
@@ -124,5 +127,56 @@ const streamEntityCsvs = (inStream, properties) => {
   return PartialPipe.of(inStream, rootStream, csv());
 };
 
+// Paging is done at the database level
+const streamEntityOdata = (inStream, properties, domain, originalUrl, query, tableCount) => {
+  const serviceRoot = getServiceRoot(originalUrl);
+  const { limit, offset, shouldCount } = extractPaging(query);
+
+  const schema = {
+    name: null,
+    label: null
+  };
+
+  for (const property of properties) {
+    schema[property.name] = null;
+  }
+
+  let isFirstEntity = true;
+  const rootStream = new Transform({
+    writableObjectMode: true, // we take a stream of objects from the db, but
+    readableObjectMode: false, // we put out a stream of text.
+    transform(entity, _, done) {
+      try {
+        if (isFirstEntity) { // header or fencepost.
+          this.push('{"value":[');
+          isFirstEntity = false;
+        } else {
+          this.push(',');
+        }
+
+        this.push(JSON.stringify(mergeLeft({
+          name: entity.uuid,
+          createdAt: entity.createdAt,
```

Comment:
`createdAt` isn't a reserved property name, is it? Should we put it in a `__system` `ComplexType` similar to what we do with submissions? I imagine that we'll continue accumulating metadata/summary properties like we have with submissions, so a `__system` object might be a good pattern to put in place now.